### PR TITLE
Fix `expose` well-formedness check

### PR DESCRIPTION
**Pull request by bifbof**
Just a small bug fix + test for a forgotten `?` in `expose` wf check.
I'm not sure where to put the tests, there's not really a structure in `minitest/src/tests`, so I created a new file and put the test in there.

### CODE REVIEW

**Comment by RalfJung** [comment]
Good catch!

> there's not really a structure in minitest/src/tests

It's a bit chaotic, yes. If you have ideas for how to clean it up, I am all ears. :)

There seem to be no existing tests for `expose` so yeah this location makes sense.

File: tooling/minitest/src/tests/expose.rs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+use crate::*;
+
+/// test if exposed called with non-pointer is ill-formed
```
Comment (RalfJung) [inline tooling/minitest/src/tests/expose.rs:3]:
```suggestion
/// Test if `expose` called with non-pointer is ill-formed.
```

File: tooling/minitest/src/tests/expose.rs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#[test]
+fn requires_pointer() {
+    let locals = [];
+    let blocks = [block!(expose(const_bool(false)), exit())];
+
+    let program = program(&[function(Ret::No, 0, &locals, &blocks)]);
+    assert_ill_formed(program);
+}
+
+/// test if exposed called with pointer works
```
Comment (RalfJung) [inline tooling/minitest/src/tests/expose.rs:13]:
```suggestion
/// Test if `expose` called with pointer works.
```

**Comment by bifbof** [comment]
@RalfJung I've updated this PR :)

**Comment by RalfJung** [comment]
Thanks!

**Pull request merged**